packages/core/src/context/MemoryStore.ts

```
@@ -0,0 +1,443 @@
+          return date;
+        }
+      }
+    }
+
+    return undefined;
+  }
+
+  /**
+   * Search for semantically similar memories
+   */
+  async searchSemanticMemories(query: string, k: number = 5): Promise<MemorySearchResult[]> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)